controller/tasks.go

```diff
@@ -241,6 +245,40 @@ func (c *Controller) getTaskHandler(w http.ResponseWriter, r *http.Request) {
 	dagjson.Encoder(task.Representation(), w)
 }
 
+func (c *Controller) carHandler(w http.ResponseWriter, r *http.Request) {
+	logger := log.With("req_id", r.Header.Get("X-Request-ID"))
+
+	logger.Debugw("handle request", "command", "car")
+	defer logger.Debugw("request handled", "command", "car")
+
+	w.Header().Set("Content-Type", "application/octet-stream")
+
+	store := c.db.Store(r.Context())
+	rootCid, err := store.Head()
+	if err != nil {
+		log.Errorw("get task DB error", "err", err.Error())
+		w.WriteHeader(http.StatusBadRequest)
+		return
+	}
+
+	w.WriteHeader(http.StatusOK)
+
+	ssb := builder.NewSelectorSpecBuilder(basicnode.Prototype__Any{})
+	ss := ssb.ExploreRecursive(selector.RecursionLimitNone(), ssb.ExploreUnion(
+		ssb.Matcher(),
```

Comment:
Non-blocking: FYI, for the purposes of CAR writing, the ExploreUnion and the Matcher are unnecessary